src/coord/src/catalog/error.rs

```diff
@@ -168,6 +173,18 @@ https://materialize.com/docs/cli#experimental-mode"#
                 r#"Experimental mode is only available on new nodes. For
 more details, see https://materialize.com/docs/cli#experimental-mode"#
             ),
+            ErrorKind::FailedMigration {
+                last_version,
+                cause,
+            } => write!(
+                f,
+                r#"Migration from catalog content version {} failed; underlying error:
+{}
+
+Please report this issue to Materialize at
+https://github.com/MaterializeInc/materialize/issues/new?labels=C-bug&template=bug.md"#,
```

Comment:
Hmm, this seems like the wrong place to add such a hint. It's a great idea, but there are probably other errors that deserve this treatment too—plus not ideal to scatter such links into error messages across the codebase. (Btw, the stable shortlink for "file a bug report" is "http://materialize.com/s/bug".)

Reply:
Let me PR something at you in a sec.

Reply:
(Er, actually, probably tomorrow.)